src/main/java/com/alyrow/gdx/particle/Particle.java

```diff
@@ -111,7 +111,7 @@ public void render(SpriteBatch batch, Camera camera, Array<PhysicForce> forces)
         sprite.setPosition(physicParticle.x, physicParticle.y);
         light.setPosition(physicParticle.x + physicParticle.width / 2, physicParticle.y + physicParticle.height / 2);
 
-        if(type == ParticleType.HALO && !sprite.getColor().equals(light.getColor())) {
+        if(type.isImage() && !sprite.getColor().equals(light.getColor())) {
```

Comment:
Need to modify that with ParticleType

Reply:
[here](https://github.com/alyrow/box2d-light-particle/commit/821224c75cb5242b2da782d11c4eb7a360247870) ✔️